packages/protocol/package.json

```diff
@@ -6,61 +6,61 @@
-    "build": "yarn build:sol && yarn build:ts",
-    "prebuild": "rm -rf ./build",
+    "console": "./scripts/bash/console.sh",
     "determine-release-version": "yarn --silent ts-node --preferTsExts ./scripts/determine-release-version.ts",
-    "prepare_contracts_and_abis_publishing": "yarn ts-node --preferTsExts ./scripts/prepare-contracts-and-abis-publishing.ts",
-    "prepare_devchain_anvil_publishing": "yarn ts-node --preferTsExts ./scripts/change-anvil-devchain-package-version.ts",
-    "validate_abis_exports": "yarn ts-node --preferTsExts ./scripts/validate-abis-package.ts",
-    "sourcify-publish": "yarn ts-node ./scripts/sourcify-publish.ts",
-    "migrate": "./scripts/bash/migrate.sh",
-    "set_block_gas_limit": "./scripts/bash/set_block_gas_limit.sh",
-    "download-artifacts": "./scripts/bash/download_artifacts.sh",
-    "init-network": "./scripts/bash/init_network.sh",
-    "upload-artifacts": "./scripts/bash/upload_artifacts.sh",
-    "upgrade": "./scripts/bash/upgrade.sh",
-    "revoke": "./scripts/bash/revoke.sh",
+    "devchain": "yarn ts-node scripts/devchain.ts",
+    "devchain:reset": "yarn devchain generate-tar .tmp/devchain.tar.gz --upto 29",
+    "ganache-dev": "./scripts/bash/ganache.sh",
+    "ganache-devchain": "./scripts/bash/ganache_devchain.sh",
+    "generate-stabletoken-files": "yarn ts-node ./scripts/generate-stabletoken-files.ts",
     "govern": "./scripts/bash/govern.sh",
-    "console": "./scripts/bash/console.sh",
-    "check-versions": "./scripts/bash/check-versions.sh",
-    "check-opcodes": "yarn ts-node scripts/check-opcodes.ts",
+    "init-network": "./scripts/bash/init_network.sh",
+    "lint": "yarn run lint:ts && yarn run lint:sol",
+    "lint:sol": "solhint --version && solhint './contracts/**/*.sol' && solhint './contracts-0.8/**/*.sol'",
+    "lint:ts": "yarn run --top-level eslint .",
     "make-release": "./scripts/bash/make-release.sh",
-    "verify-deployed": "./scripts/bash/verify-deployed.sh",
-    "verify-release": "./scripts/bash/verify-release.sh",
+    "migrate": "./scripts/bash/migrate.sh",
+    "prepare-contracts-and-abis-publishing": "yarn ts-node --preferTsExts ./scripts/prepare-contracts-and-abis-publishing.ts",
+    "prepare-devchain-anvil-publishing": "yarn ts-node --preferTsExts ./scripts/change-anvil-devchain-package-version.ts",
+    "revoke": "./scripts/bash/revoke.sh",
+    "set-block-gas-limit": "./scripts/bash/set_block_gas_limit.sh",
     "size:onchain": "./scripts/bash/get_smart_contract_size_from_onchain_address.sh",
     "size:artifacts": "./scripts/bash/get_smart_contract_size_from_build_artifacts.sh",
-    "ganache-dev": "./scripts/bash/ganache.sh",
-    "ganache-devchain": "./scripts/bash/ganache_devchain.sh",
+    "sourcify-publish": "yarn ts-node ./scripts/sourcify-publish.ts",
+    "submodules:pull": "git submodule update --init --recursive",
+    "submodules:delete": "rm -rf $(git submodule | awk '{ print $2 }')",
```

Comment:
I'm not biggest fan of submodules -> maybe there is room to ensure they are installed as packages in package.json and just ignored within git? Does it make sense?

Reply:
Fully agreed (submodules are pretty terrible), but unfortunately we will most probably not be able to get rid of submodules since they are used by foundry with official dependency handling: https://book.getfoundry.sh/projects/dependencies

Reply:
I believe combination of package.json and remappings might be good starting point to at least try 😎

Reply:
Left potential idea to remove submodules to "Tasks to improve Devex", resolving this one 😄